.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
+    "cwtools.errors.ignore": [
+        "CW262",
+        "CW263",
+        "CW240",
+        "CW242",
+        "CW249"
+    ]
+}
```

Comment:
I would rather not include vscode settings in here. In fact, it would be best to just have .vscode in the gitignore in the first place.